index.md

```diff
@@ -37,6 +37,19 @@ helm install <your-relase-name> openebs-cstor/cstor --create-namespace
 helm install <your-relase-name> openebs-cstor/cstor
 ```
 
+## Dependencies
+
+By default this chart installs additional, dependent charts:
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://openebs.github.io/node-disk-manager | openebs-ndm | 1.0.2 |
+
+
+To disable the dependency during installation, set `openebsNDM.enabled` to `false`.
```

Comment:
@shubham14bajpai  lets add the full command how to use set command for reference

Reply:
something to the effect of `helm install ... --set openebsNDM.enabled=false`